source/administration/changelog.md

```diff
@@ -4,6 +4,106 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.10
+
+Release Date 2019-04-16
+
+### Highlights
+
+#### Infinite One Directional Scrolling
+ - Removed the “Load more messages” button at the top of the screen to make it easier to read messages.
+ 
+#### Interactive Ephemeral Messages
+ - Added ability to modify message attachments, allowing for richer integrations.
+
+#### Configuration in Database
+ - Added support for managing configuration options directly in the database. This option honors the current ``config.json`` to maintain backwards compatibility while providing a way to easily manage configuration settings from the system console in more complex environments.
+
+#### Elasticsearch autocomplete
+ - Added a new flag that supports autocompletion for Elasticsearch queries. When the flag is enabled, Elasticsearch will use indexed data to determine the autocompletions for users and channels.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added ability to use "c" and "sh" for code block syntax highlighting.
+ - Words that trigger mentions now supports Chinese.
+ - Added support for rendering emojis and hyperlinks in message attachment titles.
+ - Added support for showing the channel name in the message box.
+ - Added support for markdown in plugin system console help text fields.
+ - Added ability to convert Excel cells to markdown table when pasting in Mattermost.
+ - Added ability to render emojis in interactive message buttons.
+ 
+#### Plugins
+ - Added plugin support for bot accounts.
+ - Created a plugin component to override file previews.
+ - Added support for plugins to create link tooltips.
+ 
+#### Bulk Import/Export
+ - Added User Preference fields in bulk export.
+ - Added ability to include direct and group message channels and their posts in bulk export.
+ - Added ability to include deactivated users in bulk import.
+ 
+#### CLI
+ - Created CLI command ``command show`` to allow seeing detailed information of a slash command.
+ - Created CLI command ``webhook show`` to allow seeing detailed information of a webhook.
+ - Created CLI command ``team rename`` to allow renaming teams.
+ - Created CLI command ``channel search`` to allow searching for channels.
+ 
+#### Administration
+ - Improved default session timeout behavour, including changing the default ``SessionLengthWebInDays`` from 30 to 180 days.
+ - Added full text search to the system console panel to easily find options in the configuration.
+ - (Advanced Permissions) Split managing emoji permissions into "create", "delete own" and "delete others".
+ - (Advanced Permissions) Added ``List_Public_Teams``, ``Join_Public_Teams``, ``List_Private_Teams`` and ``Join_Private_Teams`` permissions.
+ - Updated Mattermost to default to console logging in a human readable format.
```

Comment:
This only affects developer mode, and not production: should we omit it from the release notes or qualify it as such?

Reply:
Removed,